pinot-core/src/test/java/com/linkedin/pinot/core/startree/TestStarTreeIntegrationTest.java

```diff
@@ -182,13 +223,18 @@ private void buildSegment(String segmentDirName, String segmentName)
         map.put(dimName, dimName + "-v" + row % (NUM_DIMENSIONS - i));
       }
 
-      // Time column.
-      map.put("daysSinceEpoch", row % 7);
+      long seed = System.nanoTime();
+      LOGGER.info("Random seed for metric values: {}", seed);
```

Comment:
Also, appending the random seed message to assert error message can reduce the size of log when the test passes (since the random seed value is useless if the test passes).